### PR TITLE
set up groups and permissions

### PR DESCRIPTION
**Pull request by fricklerhandwerk**

### CODE REVIEW

File: src/website/webview/apps.py [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+
+    def ready(self):
+        pass
+
+
+@receiver(post_migrate)
+def ensure_groups(sender, **kwargs):
+    from django.contrib.auth.models import Group
+
+    secteam, created = Group.objects.get_or_create(name="Security team")
```
Comment (RaitoBezarius) [inline src/website/webview/apps.py:18]: Why do we make dynamic groups? This feel overly complicated? All our groups are statically known: superadmins, security team, committers, maintainers, viewers. Do you want to bring more groups than those?

**Review by RaitoBezarius — CHANGES_REQUESTED** [review]
I am quite meh about the packaging changes for Python (I spent some time on fixing the mess that was happening out there.) and I do not understand the need of the extra dependency.

**Comment by RaitoBezarius** [comment]
Ah I think I fucked up my commits and the missing commit for you is here: https://github.com/Nix-Security-WG/nix-security-tracker/pull/59/commits/67dd36db418ca486afc66afa235d9f45d0b67f2f.

File: pkgs/python/django-pgpubsub/default.nix [hunk 2]
```diff
@@ -1,5 +1,4 @@
-{ lib, buildPythonPackage, fetchFromGitHub, poetry, poetry-core, django
-, django-pgtrigger }:
+{ lib, buildPythonPackage, fetchFromGitHub, python3 }:
```
[the remark this answers is not in the document]
Comment (fricklerhandwerk) [reply pkgs/python/django-pgpubsub/default.nix:1]: Right. Makes a few things simpler. I just figured that the way we build `callPackage` is weird and we should stick to the regular Nixpkgs structure, but it's probably overthinking it.

**Comment by RaitoBezarius** [comment]
The important part for permissions will be the REST API we will build, we have https://www.django-rest-framework.org/api-guide/permissions/ for that.

**Comment by RaitoBezarius** [comment]
For views, we already have the built-in system in https://docs.djangoproject.com/en/5.0/topics/auth/default/#permissions-and-authorization.

**Comment by alejandrosame** [comment]
Groups and permissions scaffold was implemented with #165 (already merged).

**Pull request closed without merging**